### DISCUSSION TITLE
Status indicator tab counter

### DISCUSSION THREAD

**Discussion by am0z** [tridactyl/tridactyl#4235] [General]
Don't know if this is an appropriate place to share custom tridactyl bits; didn't find reddit for this.

Anyway, might be useful to someone else. Also improvements are welcome.

### chrome/userContent.css
```
.TridactylStatusIndicator::before {
    content: attr(tabs)"\a0";
}
```
```
:autocmd TabEnter .* composite js tri.browserBg.tabs.query({windowId: TRI_FIRED_MOZ_WINID}).then(tabs => `${TRI_FIRED_TRI_TABINDEX}/${tabs.length}`) | js -p document.querySelector('.TridactylStatusIndicator').setAttribute('tabs', JS_ARG)
```

Shortcomings:
* it's hacky
* updates only on switching tabs; new tabs won't get it; maybe some other edge cases.

**Comment by am0z** [comment]
![2022-05-16-093031_161x51_scrot](https://user-images.githubusercontent.com/11426371/168534205-fd78296c-45f9-4124-b9ff-8598358399ad.png)

**Reply by bovine3dom** [reply]
This is really cool! I think the Wiki - https://github.com/tridactyl/tridactyl/wiki - is probably the best place for this sort of thing :)
